src/Uno.UI/ts/WindowManager.ts

```diff
@@ -565,8 +565,12 @@ namespace Uno.UI {
 			for (let i = 0; i < params.Pairs_Length; i += 2) {
 				const key = pairs[i];
 				const value = pairs[i + 1];
-
-				elementStyle.setProperty(key, value);
+				if (this.isStyleFontFamily(key)) {
+					elementStyle.setProperty(key, this.addQuotationMarks(value));
+				}
+				else {
+					elementStyle.setProperty(key, value);
+				}
```

Comment:
This feels like a performance issue. If that's the case, we'll need to change the style font family setter to be an explicit method for changing fonts through the TS Interop. What do you think @carldebilly ?

Reply:
I was thinking the same thing...  the "adjustment" should not be done in the managed side, but in TS/JS.

Reply:
@CesarRabelo You'll need to call a new method for changing the font here:
https://github.com/unoplatform/uno/blob/5e0c3818f1badff1f148325e7c3ff7b53647d4a1/src/Uno.UI/UI/Xaml/UIElement.TextHelper.wasm.cs#L93

Then declare it the same class as this method:
https://github.com/unoplatform/uno/blob/5e0c3818f1badff1f148325e7c3ff7b53647d4a1/src/Uno.UI/UI/Xaml/WindowManagerInterop.wasm.cs#L760

You can build a method similar to it.

In typescript, you'll need to create methods similar to these:
https://github.com/unoplatform/uno/blob/76f6ce75fad58e676e56c55dfafeafe73c7078d8/src/Uno.UI/ts/WindowManager.ts#L354-L377

But named `setFontFamily`.

This way, the invoking the method will cost less than when used in general by the `SetStyles` method.